src/components/BottomNav/index.tsx

```diff
@@ -0,0 +1,35 @@
+import * as S from "./bottomNav.styled";
+import { PATH } from "../../constants/path";
+import { useLocation } from "react-router-dom";
+
+const BottomNav = () => {
+  const location = useLocation();
+
+  const navList = [
+    { id: 1, name: "홈", path: PATH.ROOT, icon: <S.NavIconHomes isActive={location.pathname === PATH.ROOT ? true : false} /> },
```

Comment:
```suggestion
    { id: 1, name: "홈", path: PATH.ROOT, icon: <S.NavIconHomes isActive={pathname === PATH.ROOT ? true : false} /> },
```
위를 변경한다면 아래도 이렇게 바뀔 것 같네요

Reply:
넵 간소화해보도록 하겠습니다!